modules/archetypes/lib/policy_definitions/policy_definition_es_deploy_diagnostics_dataexplorercluster.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "Deploy-Diagnostics-DataExplorerCluster",
+  "name": "Deploy-Diagnostics-DataExplorerCLuster",
```

Comment:
Need to fix name at source in [Azure/Enterprise-Scale](https://www.github.com/Azure/Enterprise-Scale)